data/contributors/organizations.json

```
@@ -489,25 +489,25 @@
       },
       {
         "username": "vmora",
-        "author_names": "Vincent Mora",
+        "author_names": "Vincent Mora,vmora",
```

```suggestion
        "author_names": "Vincent Mora",
```

username and author_names will be concatenated automatically